### ISSUE TITLE
You're attempting to include a mixin that is either null or not an object

### ISSUE THREAD

**Issue by gkrinc** [aksonov/react-native-tableview#120]
When importing:

`import TableView from 'react-native-tableview';`

I'm getting this warning:

`ExceptionsManager.js:82Warning: ReactClass: You're attempting to include a mixin that is either null or not an object. Check the mixins included by the component, as well as any mixins they include themselves. Expected object but got undefined.`

**Versions**

- react-native: 0.35.0
- react-native-tableview: 1.6.0

**Comment by roysG** [comment]
+1

**Comment by razvan-paul** [comment]
+1

**Comment by aksonov** [comment]
Does it work with earlier RN versions? @nyura123 could you check?

**Comment by iRoachie** [comment]
No progress on this?

**Comment by chetstone** [comment]
@aksonov I've had this issue with RN 0.32 and 0.33. I worked around it by [removing the require](https://github.com/chetstone/react-native-tableview/commit/f0ba7994f5e620a7e1c5a01bce80e45353de5a17) of `NativeMethodsMixin`, which is apparently unused.

Weirdly, though, yesterday I cloned my project to another machine and that machine doesn't need that workaround, it works with b85f0d4924bafe3ef16cc0d6a7f4ae564435653e. I cannot figure out why. The code should be identical on both machines. The new project had a new install of all node modules, but when I removed and reinstalled all node modules on the old machine, it didn't change the behaviour --- it still needs the workaround.

I see [another project had to remove](https://github.com/airbnb/react-native-maps/pull/511) NativeMethodsMixin but then they had to add it back in. Go figure.
